doc/man3/SSL_CTX_set_num_tickets.pod

```diff
@@ -0,0 +1,58 @@
+=pod
+
+=head1 NAME
+
+SSL_set_num_tickets,
+SSL_get_num_tickets,
+SSL_CTX_set_num_tickets,
+SSL_CTX_get_num_tickets
+- control the number of TLSv1.3 session tickets that are issued
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_set_num_tickets(SSL *s, size_t num_tickets);
+ size_t SSL_get_num_tickets(SSL *s);
+ int SSL_CTX_set_num_tickets(SSL_CTX *ctx, size_t num_tickets);
+ size_t SSL_CTX_get_num_tickets(SSL_CTX *ctx);
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_num_tickets() and SSL_set_num_tickets() can be called for a server
+application and set the number of session tickets that will be sent to the
+client after a full handshake. Set the desired value in the B<num_tickets>
+argument. Typically these functions should be called before the start of the
+handshake. They may be called later to control the number of tickets that will
+be issued on receipt of a post-handshake certificate from the client (after an
+initial full handshake). The default number of tickets is 2. The default number
```

Comment:
I would remove the word `default` from the sentence about resumption ticket issuance. Then we may not even need to say this can't be changed, but perhaps clearer if emphasised.

Reply:
Oops, actually, since it can be changed to 0 by callbacks, my comment is wrong... It is 1 if any tickets are issued, and that can't be changed, but issuing of tickets after resumption can be turned off by callbacks... Amend as you see fit...